core/src/OCP/accessibility.js

```diff
@@ -22,11 +22,23 @@
 
 import { loadState } from '@nextcloud/initial-state'
 
+/**
+ * Set the page heading
+ *
+ * @param {string} [heading] page title from the history api
```

Comment:
```suggestion
 * @param {string} heading page title from the history api
```

https://jsdoc.app/tags-param.html